test/e2e/router-mgmt/qdmanage.go

```diff
@@ -0,0 +1,89 @@
+package router_mgmt
```

Comment:
Suggest moving this directory under `framework` and renaming to qdr-mgmt, qdr-utils or some other qdr prefix...

Reply:
Maybe this should also address the first suggestion for ListNodesForPod. I will move it and update you once done for a new review.